docs/program-incentives/ark-grants-program-guidelines.md.blade.php

```diff
@@ -70,13 +70,13 @@
 
 ### Submission Process
 
-Applications should be completed using the [**ARK Grants Application Template** ](ark-grants-program-guidelines#ark-grants-application-template)below. Please fill out all information as requested, to include your personal contact information for coordination of your application. Project reviews may take up to 14 days for initial review of your application. During this time, the ARK.io team will reach out to you with any questions or comments on your application. 
+Applications should be completed using the [ARK Grants Application Template](#ark-grants-application-template) below. Please fill out all information as requested, to include your personal contact information for coordination of your application. Project reviews may take up to 14 days for initial review of your application. During this time, the ARK.io team will reach out to you with any questions or comments on your application. 
 
 Once the project review is completed, we will notify you with an official decision about your project, including any feedback we may have. If your project is approved, we will provide instructions on how to proceed at that time. 
 
 Thank you for your interest in the ARK Grants program and we look forward to reading your proposals!
 
- **Please submit the completed application to** [**Grants@ARK.io**](mailto:Grants@ARK.io) **with the title "**_**ARK Grants Application**_**".**
+ **Please submit the completed application to** [Grants@ARK.io](mailto:Grants@ARK.io) **with the title "**_**ARK Grants Application**_**".**
```

Comment:
if you remove all the `**` being opened and closed throughout the sentence and only put it at the start and end, it should make everything including the url component bold 🤔